### PR TITLE
added page components

### PR DESCRIPTION
**Pull request by CampbellSDC**
This was quick and dirty. I'll go back on my lunch to get this updated, because the pages aren't rendering. I'll figure that out. It's at least not throwing a bunch of errors...which also sucks lol

### CODE REVIEW

File: client/src/App.jsx [hunk 1]
```diff
@@ -1,34 +1,27 @@
...
-          Edit <code>src/App.jsx</code> and save to test HMR
-        </p>
-      </div>
-      <p className="read-the-docs">
-        Click on the Vite and React logos to learn more
-      </p>
-    </>
+    <BrowserRouter>
+      <Routes>
+        <Route element={<Layout />}>
```
Comment (raygdev) [inline client/src/App.jsx:15]: Ok, so add the path to this one
```javascript
<Route path="/" element={<Layout />} >
```
This is where all the child routes should get routed through the layout outlet. You can set index as a property on the home page route.

File: client/src/App.jsx [hunk 2]
```diff
@@ -1,34 +1,27 @@
...
+
+          <Route path='/' element={<Home />} />
+          <Route path='/about' element={<About />} />
```
Comment (raygdev) [inline client/src/App.jsx:18]: We can remove any leading `/` and just pass the name of the route. 
```javascript
<Route path="about" element={ <About />} />
```
Do the same for login.
Then revist the page.

File: client/src/App.jsx [hunk 3]
```diff
@@ -1,34 +1,27 @@
...
-        </button>
-        <p>
-          Edit <code>src/App.jsx</code> and save to test HMR
-        </p>
-      </div>
-      <p className="read-the-docs">
-        Click on the Vite and React logos to learn more
-      </p>
-    </>
+    <BrowserRouter>
```
Comment (raygdev) [inline client/src/App.jsx:13]: Remove the browser router here and only return the routes. Wrap the App component in `main.jsx`

**Review by raygdev — CHANGES_REQUESTED** [review]
Review the suggestions and see how this works for you. If it doesn't I will pull in the changes later. I don't think there is a need to create another issue for the breaks created here. We can work through the breaking changes in the current issue branch we are working on and resolve the issue that way.

**Comment by raygdev** [comment]
It's  likely from the import of the Login component since it doesn't exist yet

**Comment by raygdev** [comment]
Also, either push the code so I csn see what we have or provide a snippet of the changes you've made so far...

**Comment by CampbellSDC** [comment]
Yup! I thought I had checked all of the components, but apparently just ignored the Login lol. Ok, we are up and running

**Comment by raygdev** [comment]
@CampbellSDC Alright cool. Push the code and let's see what we've got!

**Review by CampbellSDC — COMMENTED** [review]
All components rendering

**Review by raygdev — COMMENTED** [review]
Looking good!

**Review by raygdev — APPROVED** [review]
Noice!

**Pull request merged**